data/consultation/medications/Hydrocortisone.json

```diff
@@ -0,0 +1,7 @@
+{
+    "drug_name": "Hydrocortisone 100 mg injection",
+    "units": "mg",
+    "frequency": "Thrice a day",
+    "dose": "3",
+    "duration": "7"
+}
```

Comment:
notes key missing in json

Reply:
I didn't add because we are not using in the script.Will update in all the json

Reply:
updated